run_nwaku.sh

```diff
@@ -33,29 +35,42 @@ else
     fi
 fi
 
+#Function to get the index of the container and use it to retrieve a private key to be used to generate the keystore
+get_private_key(){
```

Comment:
and here we can direactly have `getPrivateKeys.sh` content. where anvil-config is loaded, parsed and where INDEX is taken.

Reply:
Done with some modifications for /bin/ash script limitations (https://github.com/waku-org/waku-simulator/pull/57/commits/71882e78b56af87b10bbba8d5f2cd3986689aff0 )